practices/java-web/HelloWorldServletMvcWebApplication/src/main/java/com/dowa/java/web/HelloWorldMvcServlet.java

```diff
@@ -21,10 +24,27 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
 
         RequestDispatcher requestDispatcher = req.getRequestDispatcher("/Hello.jsp");
+        RequestDispatcher requestDispatcher1 = req.getRequestDispatcher("/Error.jsp");
 
+        boolean isStudent = false;
         String name = req.getParameter("name");
-        req.setAttribute("fakeName", StringUtils.reverse(name));
+        ArrayList names = new ArrayList();
+        //req.setAttribute("fakeName", StringUtils.reverse(name));
+
+        names.add("Nexuz");
+        names.add("Nana");
+        names.add("Bruce");
+        names.add("Mary");
+        names.add("2Pac");
```

Comment:
try using Arrays.asList("str1", "str2") instead or like  names = new ArrayList(new String[] {"str1", "str2"});

I prefer the first option